src/main.lua

```diff
@@ -1102,96 +1105,68 @@ addEventingHandler(ActionMap.TokenCost, utils.hasMatchingTag("Action", ActionMap
 			from = msg.From,
 		}
 	)
-	if not shouldContinue2 then
+	if not shouldContinue2 or not tokenCostResult then
```

Comment:
also had this there for awhile, but was thinking it'd be nice if we could make the type def of `eventingPCall` handle this the same way `pcall` does. in TS its like `returns [false, undefined] | [true, T: tokenCostResult]`.